s/pbchunk.ss

```diff
@@ -837,65 +846,73 @@
          (define (done)
            (next))
 
-         (define (pre)
-           (string-append
-            (format "/* 0x~x */ " i)
-            (if (>= i start-i) "  " "/* ")))
-         (define (post)
-           (if (>= i start-i) " " " */ "))
+         (define (emit-pre)
+           (emit-c! o "/* 0x~x */" i)
+           (unless (>= i start-i)
+             (emit-c! o " /*")))
+         (define (emit-post)
+           (unless (>= i start-i)
+             (emit-c! o " */")))
+
+         (define (emit-desc fmt . args)
+           (apply emit-c! o fmt args))
```

Comment:
I think these names should be suffixed with `!` for consistency with `emit-c!`

Reply:
Good point about the inconsistency. Do you think it would be ok to instead drop the `!` on `emit-c!`? The `emit-c!` function is replacing `fprintf` here, and there are many more `emit` functions in the pbchunk implementation that were originally about writing to a port. That is, I put a `!` on `emit-c!` because it ends up at `set-box!`, but I'm now thinking I should have left off the `!` because it's implementing something like a port.

Reply:
That would work for me, too.

Reply:
Ok, done!